LayoutTests/webrtc/multi-video.html

```diff
@@ -65,6 +65,11 @@
                 firstConnection.addTrack(track4, newStream);
                 firstConnection.addTrack(track5, newStream);
                 firstConnection.addTrack(track6, newStream);
+
+                // Restrict each transceiver to one codec (H.264) to prevent payload types from overflowing in SDP offer.
+                var codecs = RTCRtpSender.getCapabilities("video").codecs;
+                var h264Codecs = codecs.filter(codec => { return codec.mimeType === "video/H264"; });
```

Comment:
You can just var, but event better if you use const or let. const in this case is enough